app/styles/modules/_input-row.scss

```diff
@@ -0,0 +1,101 @@
+html[dir='ltr'] {
+  input[type='email'],
+  input[type='text'],
+  input[type='password'] {
+      @include formElement();
+      padding: 0 0 0 $input-left-right-padding;
+  }
+}
+
+html[dir='rtl'] {
+  input[type='email'],
+  input[type='text'],
+  input[type='password'] {
+      @include formElement();
+      padding: 0  $input-left-right-padding 0 0;
+  }
+}
+
+.input-row {
+  margin-bottom: 12px;
+  position: relative;
+  width: 100%;
+
+  &:hover input {
+    border-color: $input-border-color;
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  button,
+  input {
+    @include font();
+  }
+
+  input[type='email'],
+  input[type='text'] {
+  }
+
+  input::-webkit-input-placeholder,
+  input:-moz-placeholder, /* Firefox 18- */
+  input::-moz-placeholder,  /* Firefox 19+ */
+  input:-ms-input-placeholder {
+    color: $input-placeholder-color;
+  }
```

Comment:
Interesting... Doesn't Autoprefixer do this magic for us? Or are we explicitly specifying 18- rules because Autoprefixer doesn't generate prefixes for that far back into the past?